kovri.md

```diff
@@ -9,76 +9,53 @@ Ring signatures obscure the sender.
-If Alice sends her transaction with Kovri, nodes will no longer see her IP address, making passive surveillance tactics impractical.
 
 Now let’s imagine a different scenario.
 Suppose Charlie wants to support the Monero network by running a full node at his home.
 After a few weeks, he receives a cease and desist letter from his Internet Service Provider informing him that running a node is a violation of the terms of service.
-Charlie could have avoided this situation in the first place by anonymizing his Internet traffic with Kovri.
-Kovri substantially improves Monero's censorship resistance by making it even more difficult to detect and shut down nodes.
 
 In our last scenario, suppose Dave is an operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
 Other nodes could purposefully reject his solved blocks to express their disagreement with his political views.
-However, if Dave can send his solved blocks through Kovri, then no one can censor the new blocks created by his pool, since the source of solved blocks are unknown.
 
-As you can see, Kovri is a software that allows users to send transactions more privately and anonymously than ever before.
-Let's next go over some fairly technical stuff to have a better understanding of how Kovri works under the hood.
-
-## 5) Kovri and Technical Attributes
-
-Kovri is currently built from I2P's open standards. I2P stands for the invisible internet project. Kovri is as an anonymizing router which tunnels traffic through the I2P network.
-
-To protect their anonymity, each participant connects to other network members and uses them to pass messages.
-In most circumstances, people will choose several peers for incoming and outgoing connections.
-People can use more peers for greater privacy at the cost of performance.
-
-Suppose Dave's mining pool has just solved a block and would like to send this transaction information to the other network members.
-Dave's router will send the message to his two outgoing peers, who will then search for the incoming peers of other nodes on the network.
-Dave can commincate securely without other Monero nodes on the network knowing what his actual IP address is.
-The other participants in the I2P network do not know what their place is in the chain of routing, nor do they know what information is being relayed inside the network.
-This is called garlic routing, wherein the only information shown to peers is the instruction set for sending to the next peer.
+Alice, Bob, Charlie, and Dave all have at least one thing in common: their IP addresses were exposed.
+Users could try to hide IP addresses with Tor or a VPN; however both of these strategies have serious weaknesses.
+The Tor network has "semi-trusted" Directory Authorities which  negatively affect decentralized applications and correlation attacks are possible with VPNs, making it easy for large attackers to de-anonymize a user's traffic.
```

Comment:
I prefer the original "which give a handful of Tor node operators overreaching influence into network consensus".  Also, if you include the the VPN argument in the same sentence, separate with a comma or even a semi-colon.  Probably semicolon, since they're both basically complete sentences.